code/modules/mob/living/carbon/human/species_types/IPC.dm

```diff
@@ -239,3 +241,9 @@
 			BP.limb_id = chassis_of_choice.limbs_id
 			BP.name = "\improper[chassis_of_choice.name] [parse_zone(BP.body_zone)]"
 			BP.update_limb()
+
+/mob/living/carbon/proc/charge(datum/source, amount, repairs)
+	if(nutrition < NUTRITION_LEVEL_WELL_FED)
+		adjust_nutrition(amount / 10) // The original amount is capacitor_rating*100
+	if(repairs)
```

Comment:
Please do not add repairing into these, IPCs are already incredibly easy to fix.